File: packages/cli/templates/stacks/mcp-python/src/prompts/summarize.py
"""Summarize prompt — returns a templated user message."""


def summarize(text: str) -> str:
    """Returns the user message body for a summarize prompt.

    Args:
        text: Text to summarize.
    """
    return f"Summarize the following text in 1-2 sentences.\n\n{text}"

File: packages/cli/templates/stacks/mcp-go/internal/prompts/summarize.go
// Package prompts holds pure prompt builders.
package prompts

import "fmt"

// Summarize returns the user-message body for a summarize prompt.
func Summarize(text string) string {
	return fmt.Sprintf("Summarize the following text in 1-2 sentences.\n\n%s", text)
}
